src/main/resources/db/changelog/v.0.2/db.changelog-form-status.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: 08042021-add-table-form-status
+      author: valentin_velichko
+      changes:
+        - createTable:
+            tableName: form_status
```

Comment:
i'm not sure that this table is required.

Reply:
Deleted